addons/stock_picking_batch/security/stock_security.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<data noupdate="0">
+    <record model="ir.rule" id="stock_picking_batch_multicompany_rule">
+        <field name="name">stock.picking.batch multi-company</field>
+        <field name="model_id" ref="model_stock_picking_batch"/>
+        <field name="domain_force">[('company_id', 'in', company_ids)]</field>
+    </record>
```

Comment:
Can you please add in `<field name="global" eval="True"/>`?

Reply:
Actually nevermind, it looks like `<field name="global" eval="True"/>` isn't added anymore. So just the file rename please :sweat_smile: